data/json/regional_map_settings.json

```
@@ -990,7 +990,8 @@
         "cs_sex_shop": 100,
         "cs_tire_shop": 100,
         "homeless_shelter": 100,
-        "parking_garage": 200
+        "parking_garage": 200,
+        "afs_augmentation_clinic_1": 400
```

This is double the weight doctor's offices normally use, so may be better to use the same value?
```suggestion
        "afs_augmentation_clinic_1": 200
```

---

Wasn't sure what weight did honestly or what to set it to, figured someone would know a bit more about it and that Aftershocks weight values were set to sane amounts.